src/features/patient/fields/BloodGroupQuestion.tsx

```diff
@@ -0,0 +1,62 @@
+import { FormikProps } from 'formik';
+import React from 'react';
+import * as Yup from 'yup';
+
+import DropdownField from '@covid/components/DropdownField';
+import i18n from '@covid/locale/i18n';
+import { PatientInfosRequest } from '@covid/core/user/dto/UserAPIContracts';
+
+export interface BloodGroupData {
+  bloodGroup: string;
+}
+
+interface Props {
+  formikProps: FormikProps<BloodGroupData>;
+}
+
+enum BloodGroupQuestionFields {
+  TYPE_A = 'a',
+  TYPE_B = 'b',
+  TYPE_AB = 'ab',
+  TYPE_O = 'o',
+  DONT_KNOW = 'dont_know',
```

Comment:
Per Somesh's comment above, I believe this should be 'unsure'?
```suggestion
  DONT_KNOW = 'unsure',
```